### PR TITLE
install Visual C++ Redistributable Packages

### PR DESCRIPTION
**Pull request by ubozov**

### CODE REVIEW

File: dapp/dapp_windows.go [hunk 1]
```diff
@@ -198,3 +198,14 @@ func Exists(role string, logger log.Logger) (*Dapp, bool) {
 	}
 	return d, true
 }
+
+func (d *Dapp) installInitialize(logger log.Logger) error {
+	// TODO (ubozov) this algorithm should be reviewed.
+	// Executes to install run-time components (the Visual C++
+	// Redistributable Packages for VS 2013) that are required
+	// to run postgresql database engine.
```
Comment (furkhat) [inline dapp/dapp_windows.go]: move this methods above func definition, like godoc comments.

File: dapp/dapp_windows.go [hunk 2]
```diff
@@ -198,3 +198,14 @@ func Exists(role string, logger log.Logger) (*Dapp, bool) {
 	}
 	return d, true
 }
+
+func (d *Dapp) installInitialize(logger log.Logger) error {
+	// TODO (ubozov) this algorithm should be reviewed.
+	// Executes to install run-time components (the Visual C++
```
Comment (furkhat) [inline dapp/dapp_windows.go]: `Installs run-time compon...`

File: dapp/dapp_windows.go [hunk 3]
```diff
@@ -198,3 +198,14 @@ func Exists(role string, logger log.Logger) (*Dapp, bool) {
 	}
 	return d, true
 }
+
+func (d *Dapp) installInitialize(logger log.Logger) error {
```
Comment (furkhat) [inline dapp/dapp_windows.go]: method name is not descriptive.

File: dapp/dapp_windows.go (hunk 1 above)
Comment (ubozov) [reply dapp/dapp_windows.go]: moved

File: dapp/dapp_windows.go (hunk 2 above)
Comment (ubozov) [reply dapp/dapp_windows.go]: fixed

File: dapp/dapp_windows.go (hunk 3 above)
Comment (ubozov) [reply dapp/dapp_windows.go]: renamed

**Pull request merged**